Sources_v3/StreamChat/Database/DTOs/AttachmentDTO.swift

```diff
@@ -65,7 +65,7 @@ class AttachmentDTO: NSManagedObject {
             return existing
         }
         
-        let new = AttachmentDTO(context: context)
+        let new = NSEntityDescription.insertNewObject(forEntityName: Self.entityName, into: context) as! AttachmentDTO
```

Comment:
Either document difference in code or at least add link to SO.

Reply:
This was a bug. We have it correctly for all other DTOs. Copy&Paste mistake basically.